### PR TITLE
Update moderators.md

### PR DESCRIPTION
**Pull request by danpopSD**
added myself to moderators per the guidelines and group instruction.

Signed-off-by: Dan Papandrea <dan.papandrea@sysdig.com>

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: communication/moderators.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Moderators seats: 10
 | Chris Short    | @chrisshort         | Americas | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
 | Taylor Dolezal | @onlydole           | Americas | [PT - Pacific Time (US West Coast)](https://time.is/PT) |
 | David McKay    | @rawkode            | EMEA     | [GMT - Greenwich Mean Time](https://time.is/GMT)        |
-| _Open_          | _Open_             |          |                                                         |
+| POP            | @POP                |Americas  | [ET - Eastern Time (US East Coast)](https://time.is/ET)                                                        |
```
Comment (mrbobbytables) [inline communication/moderators.md:157]:
```suggestion
| Dan “POP” Papandrea | @danpopSD      |Americas  | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
```
We do need this to be accurate for record keeping 👍

**Comment by jeefy** [comment]
Or
/lgtm
lol

**Pull request merged**